Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Use Ubuntu 14.04 as the base container
+FROM ubuntu:14.04
+
+MAINTAINER "Robert McDermott" robert.c.mcdermott@gmail.com
+
+# ADD the CRAN Repo to the apt sources list
+RUN echo "deb http://cran.fhcrc.org/bin/linux/ubuntu trusty/" > /etc/apt/sources.list.d/cran.fhcrc.org.list
+# Add the package verification key
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 51716619E084DAB9
+
+# Update the system and install packages
+RUN apt-get -y update && apt-get -y install \
```

Comment:
@LisaMc yes I'll be updating the install documentation soon to cover platform specific instructions. For folks want to deploy Oncoscape via container, they don't need to worry about dependencies as they are all contained in the docker container.